irohad/ametsuchi/impl/redis_flat_block_query.hpp

```diff
@@ -46,6 +54,13 @@ namespace iroha {
       std::vector<uint64_t> getBlockIds(const std::string &account_id);
 
       /**
+       * Returns block id which contains transaction with a given hash
+       * @param hash - hash of transaction
+       * @return block id or boost::none
+       */
+      boost::optional<uint64_t> getBlockId(const std::string &hash);
```

Comment:
As I remember block id have own typedef. If it is not true. Need to make typedef for it.